src/OpenTK/Toolkit.cs

```diff
@@ -152,10 +152,30 @@ public static Toolkit Init(ToolkitOptions options)
                          * NOTE:
                          * Non-Windows platforms should be handled via the OpenTK.dll.config file as appropriate
                          */
-                        string path = Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
-                        path = Path.Combine(path, IntPtr.Size == 4 ? "x86" : "x64");
-                        bool ok = SetDllDirectory(path);
-                        if (!ok) throw new System.ComponentModel.Win32Exception();
+                        string assemblyLocation = Assembly.GetEntryAssembly().Location;
+						if(assemblyLocation != null)
+						{
+							string path = Path.GetDirectoryName(assemblyLocation);	
+							path = Path.Combine(path, IntPtr.Size == 4 ? "x86" : "x64");
+							try
+							{
+								bool ok = SetDllDirectory(path);
+                        		if (!ok) 
+								{
+#if DEBUG
```

Comment:
It's unfortunate but nuget doesn't allow the downstream user to switch between debug and release configurations, so users of OpenTK will never get these exceptions if they are using the OpenTK github package.

The easiest solution to this issue is to just not `#if` define these out.